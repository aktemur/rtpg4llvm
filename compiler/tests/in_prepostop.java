/* expr */
*y + &x * *col++ + --x * &x[i]  ; 
