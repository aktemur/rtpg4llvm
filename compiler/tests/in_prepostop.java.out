mkBinOp(Plus,
         mkBinOp(Plus,
            mkUnaryOp(Star, new VarExpr("y")),
            mkBinOp(Star,
               mkUnaryOp(Ampersand, new VarExpr("x")),
               mkUnaryOp(Star, mkUnaryPostOp(PlusPlus, new VarExpr("col"))))),
         mkBinOp(Star,
            mkUnaryOp(MinusMinus, new VarExpr("x")),
            mkUnaryOp(Ampersand, new ArrayExpr(new VarExpr("x"), new VarExpr("i")))))
